nextflow.config

```diff
@@ -48,12 +48,13 @@ params {
     group_by_umi_strategy = 'Adjacency' // default strategy when running with UMI for GROUPREADSBYUMI
 
     // Preprocessing
-    aligner            = 'bwa-mem'  // Default is bwa-mem, bwa-mem2 and dragmap can be used too
-    use_gatk_spark     = null       // GATK Spark implementation of their tools in local mode not used by default
-    save_mapped        = false      // Mapped BAMs not saved
-    save_output_as_bam = false      // Output files from preprocessing are saved as bam and not as cram files
-    seq_center         = null       // No sequencing center to be written in read group CN field by aligner
-    seq_platform       = 'ILLUMINA' // Default platform written in read group PL field by aligner
+    aligner                       = 'bwa-mem'  // Default is bwa-mem, bwa-mem2 and dragmap can be used too
+    use_gatk_spark                = null       // GATK Spark implementation of their tools in local mode not used by default
+    save_mapped                   = false      // Mapped BAMs not saved
+    save_output_as_bam            = false      // Output files from preprocessing are saved as bam and not as cram files
+    seq_center                    = null       // No sequencing center to be written in read group CN field by aligner
+    seq_platform                  = 'ILLUMINA' // Default platform written in read group PL field by aligner
+    markduplicates_pixel_distance = 100        // Default value for GATK MarkDuplicates
```

Comment:
somehow need to make sure we check tool updates when the underlying defaults change. If we don't set the value here at all it should be using the default, correct? 

That might be safer to just use the existing default